skills/lark-workflow-okr-review/SKILL.md

```
@@ -0,0 +1,127 @@
+## 前置条件
+
+仅支持 **user 身份**。执行前确保已授权：
+
+```bash
+lark-cli auth login --domain okr        # 基础（查看 OKR + 进展）
+lark-cli auth login --domain okr,task   # 含关联任务（推荐）
+```
+
+## 工作流
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)